DigitalLearningSolutions.Web/Helpers/LoginClaimsHelper.cs

```diff
@@ -86,6 +85,26 @@ UserAccount userAccount
             return claims;
         }
 
+        private static IEnumerable<Claim> GetClaimsForUserCentre(
+            AdminAccount? adminAccount,
+            DelegateAccount? delegateAccount
+        )
+        {
+            return adminAccount == null && delegateAccount == null
+                ? new List<Claim>()
+                : new List<Claim>

```

Comment:
What would be a better way to deal with this? If I remember correctly, I wrote it this way not because I needed to code for a specific scenario, but because I wanted to do something explicit[ly bad] in the situation where both of the nullable inputs are null, since this is a situation that the code currently allows for, even if it will actually never happen.

Reply:
I have made changes the the `UserCentreAccounts` model such that this code is now
```
return centreAccounts.CanLogInToCentre? new List<Claim>
                {
                    new Claim(CustomClaimTypes.UserCentreId, centreAccounts.CentreId.ToString()),
                    new Claim(CustomClaimTypes.UserCentreName, centreAccounts.CentreName),
                }
                : new List<Claim>();
```

Reply:
I have changed this method so that it throws an exception when there are no active, approved accounts